### DISCUSSION TITLE
How to reorganize tokens?

### DISCUSSION THREAD

**Discussion by eried** [akopachov/flipper-zero_authenticator#99] [General]
Is it possible to change the index?

**Comment by akopachov** [comment]
Yes, it is possible. There is CLI command `totp move <index> -i <new_index>` or `totp mv <index> -i <new_index>` it accepts existing token index and new token index. By using it it is possible to reorganize tokens as needed. 

Also you may reorganize tokens by manually editing `/ext/authenticator/totp.conf` file, reorganizing tokens there as needed. However I suggest to  use CLI to do that to minimize the risk of corrupting conf file.

**Comment by eried** [comment]
oh thanks! I think the cli help has a small mistake because it says `-i <new_index>` is optional in the help,  but is not

**Reply by akopachov** [reply]
Good catch! Will make a change to make this command look like this `totp mv <index> <new_index>`. Thanks for catching this issue

**Reply by akopachov** [reply]
Logged issue #102

**Reply by eried** [reply]
yeah I was getting an error (I though with 1 index was just saying -> move this on top) but the error message was unclear. New command is way better like that
